### ISSUE TITLE
[Good first PR] Clean up examples

### ISSUE THREAD

**Issue by bernie-g** [bernie-g/geckolib#21]
Hey, since GeckoLib is an official sponsor of Modtoberfest, I figured I'd setup a few good issues people can work on.

This one should be pretty easy. Basically we need to add more in-depth and commented examples to GeckoLib so people have a better reference.

Good example ideas:
- An entity with several attacks on a timer
- An entity with several animation controllers running multiple animations simultaneously
- An entity with many animations such as walking, flying, idle, sleeping, eating, attacking, and a death animation
- A tile entity animation with several animations
- Basically anything creative that makes good use of geckolib's features and is preferably commented 

Make sure you're on the [develop ](https://github.com/bernie-g/geckolib/tree/develop)branch

Thank you for contributing and using geckolib!
